app/views/gobierto_plans/plan_types/show.html.erb

```diff
@@ -37,6 +37,7 @@
     <div id="gobierto-planification" class="gobierto_planification" 
       data-baseurl="<%= gobierto_plans_plan_path(slug: @plan.plan_type.slug, year: @year) %>"
       data-plan-id="<%= @plan.id %>" />
+    </div>
```

Comment:
This is wrong. Look at the end of the line `data-plan-id="<%= @plan.id %>" />`
The slash means the `div` at line 37 is closed with no content.
So the line 40 breaks the markup, the closing div is pointless there. Better open a new one in line 40, and leave the closing at line 49 as it is